data/presets/presets/amenity/toilets/disposal/portapotty.json

```diff
@@ -0,0 +1,31 @@
+{
+    "icon": "tnp-2009541",
+    "fields": [
+        "toilets/disposal",
+        "{amenity/toilets}"
+    ],
+    "moreFields": [
+        "{amenity/toilets}"
+    ],
+    "geometry": [
+        "point",
+        "area",
+        "vertex"
+    ],
+    "terms": [
+        "restroom",
+        "privy",
+        "portable toilet",
+        "chemical toilet"
+    ],
+    "tags": {
+        "amenity": "toilets",
+        "toilets:disposal": "chemical",
+        "portable": "yes"
+    },
```

Comment:
Could use make this the `addTags` property and just use `amenity=toilets` + `portable=yes` for `tags`? This will mean the `toilets:disposal=chemical` tag won't be required to match features to the preset, but will be added when the feature is created.